google/ggrc-core
# GGRC-5615 Rename 'State' field into 'Launch Status' field for scoping objects

# Requirements

1. Need to re-name 'State' field into 'Launch Status' field for all scoping objects.
2. Make sure that a user is able to search scoping objects by  'Launch Status' field.

**Full list of scoping objects:**
Systems
Products
Processes
Markets
Data Assets
Facilities
Org Groups
Metrics
Product Groups
Technology Environments
Vendors
Access Groups
Projects

# Steps to test the changes

Look on tree-view and modals of mentioned objects.
Check that filters work correctly.
Check Import templates.

# Solution description

**FE part:**
Remove 'State' attr from cacheable.
Add 'Launch Status' tree-view attr for scoping models.
Add 'State' tree-view attr for other models.
Change label 'State' on 'Launch Status' on modals for scoping objects.
Change label 'State' on 'Launch Status' in advance-search-filter-state component if it is scope object.

**BE part:**
Add ScopeObject mixin with renamed state field.

# Sanity checklist




# PR Review checklist
